app/services/scheduling_service.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class SchedulingService
+  DEFAULT_TIME = "9:00am"
+  DEFAULT_DATE = Time.zone.now.tomorrow
+
+  def self.time_options(selected = nil)
+    options = times.map { |time| { text: time, value: time } }
+    if selected
+      options.each do |option|
+        if option[:text] == selected
```

Comment:
Could you lose the `if` by doing `== (selected || DEFAULT_TIME)`?

Reply:
I think we could end up with a scenario where two options are selected if the default time is found before the selected time.

Reply:
I should be fine - doing `(selected || DEFAULT_TIME)` will consistently use `selected` if it's defined, so there's no chance of accidentally choosing the `DEFAULT_TIME` if `selected` was passed in.

Reply:
Done 👍